app/views/submissions/new.html.erb

```diff
@@ -0,0 +1,2 @@
+<h2><%=t('submissions.actions.new')%></h2>
```

Comment:
I knew that but hadn't connected it as the reason for the convention around key naming in the yml files. Thanks!